scripts/chocolatey.bat

```diff
@@ -1 +1,4 @@
-powershell -NoProfile -ExecutionPolicy unrestricted -Command "iex ((new-object net.webclient).DownloadString('https://chocolatey.org/install.ps1'))"
\ No newline at end of file
+# Use TLS 1.2
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
```

Comment:
This is PowerShell code, but the script is still an old .bat cmd shell script. Maybe it's time to convert it to a full .ps1 script, but then all the packer templates must be adjusted to run it as a PowerShell script. 😅

Reply:
Oh woops 🤦‍♂ , thanks for taking a look, I'll look into how to modify this for use with a .bat file.